src/condor_starter.V6.1/jic_shadow.cpp

```diff
@@ -1396,9 +1397,10 @@ JICShadow::initUserPriv( void )
 
 	std::string owner;
 	if( run_as_owner ) {
-		if( job_ad->LookupString( ATTR_OWNER, owner ) != 1 ) {
+		if( job_ad->LookupString( ATTR_OS_USER, owner ) == false &&
+		    job_ad->LookupString( ATTR_OWNER, owner ) == false ) {
 			dprintf( D_ALWAYS, "ERROR: %s not found in JobAd.  Aborting.\n", 
-			         ATTR_OWNER );
+			         ATTR_OS_USER );
```

Comment:
misleading message, it can only print when Owner is missing.  was that intended?

Reply:
Not intentional. I'll fix it.